### ISSUE TITLE
Display more trust information

### ISSUE THREAD

**Issue by AdrianVollmer** [BloodHoundAD/BloodHound#241]
First of all, let me say that BloodHound is an incredibly useful tool, many thanks for that!

Second, let me make a feature request :)
When displaying the trusts ('map domain trusts'), I'd like to know if the trusts are transitive and what kind of trust it is. Or at least if SID filtering is active on the trust. Maybe use dashed lines, or different colors, or something like that.

What do you think?

**Comment by jesseflorig** [comment]
Does this data exist? If it has to be inferred, I can help implement the logic.

**Comment by AdrianVollmer** [comment]
Yes it does, it's already contained in the JSON file last time I checked. The attributes are called `IsTransitive` and `TrustType`
